doc/source/io.rst

```diff
@@ -1994,6 +1994,18 @@ into a flat table.
 .. ipython:: python
 
    from pandas.io.json import json_normalize
+   data = [
+             {'id': 1, 'name': {'first': 'Coleen', 'last': 'Volk'}},
+             {'name': {'given': 'Mose', 'family': 'Regner'}},
+             {'id': 2, 'name': 'Faye Raker'}
+          ]
+   json_normalize(data)
+
+.. _io.jsonl:
```

Comment:
This was to break the examples into two blocks which seem to be cleaner after checking with @DGrady. I can combine them into the same block if that's preferred